bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -44,6 +45,10 @@ public class PanelImpl extends AbstractContainerImpl implements Panel {
     @JsonIgnore
     protected boolean wrapData;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @Default(booleanValues = false)
+    protected boolean readOnly;
```

Comment:
it should be:
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Nullable
Boolean readOnly

Reply:
Hi @sakshi-arora1 ,
Why should it be nullable. Be default it should be false.

Reply:
because defaults should come from core not form model json definition

Reply:
done